src/main/java/org/bytedeco/javacv/Frame.java

```diff
@@ -221,6 +231,18 @@ public Frame clone() {
             ((Pointer[])newFrame.opaque)[1] = cloneBufferArray(samples, newFrame.samples);
         }
 
+        // Other data streams
+        if (data != null) {
+            newFrame.data = ByteBuffer.allocate(data.capacity());
```

Comment:
Don't allocate a new `ByfeBuffer` like that. Let's just use `cloneBufferArray()` here as well for consistency.

Reply:
As in turn `data` into an array that would be kept as size 1 such that `cloneBufferArray()` can be used?  At the minimum it appears the ByteBuffer.allocate call is redundant with the following lines so it will be removed.

Reply:
Yes, with an array of 1. Look the lines just above. They do exactly that.

Reply:
Done, pushed

Reply:
We don't need to change the field to an array, we just need to create a temporary local array!

Reply:
Backed out and fixed